tommath.h

```diff
@@ -23,6 +23,12 @@
 extern "C" {
 #endif
 
+/* Check for a modern-ish MacOS that supports IEEE-754 floating point numbers */
+#if    ( (defined __APPLE__) && (defined __MACH__) \
+    && ( (defined __DBL_HAS_DENORM__) && ( defined __DBL_HAS_INFINITY__) ) )
+#   define __STDC_IEC_559__ 1
+#endif
```

Comment:
I would avoid overwriting standard macros. I suggest you define a MP_HAS_NICE_FLOATING_POINT and use it everywhere.

Reply:
Oh, I forgot to check if `__STDC_IEC_559__` is already defined before.

But regarding your actual question: Using a local macro instead of one with a well defined meaning? A set `__STDC_IEC_559__` means that we have IEEE-754 compliance and that's what I want (even though I need only a small subset) and not "a nice floating point".

Moved a bit more into tommath.h but kept `__STDC_IEC_559__`. Can be easily replaced now, so if you still insist on `MP_HAS_NICE_FLOATING_POINT`&hellip;

Reply:
My point is: You should not overwrite or defined a macro which is usually defined by the compiler or by libc. And here you are defining `__STDC_IEC_559__`. Instead a custom macro should be used.